tests/test_api.py

```diff
@@ -759,9 +757,10 @@ def test_if_remove_with_remote_repo_then_clone_and_push():
                     repo=tests.resources.SAMPLE_REMOTE_REPO_URL, name="segment"
                 )
 
-    mocked_git_push.assert_called_once_with(
-        repo=MockedTemporaryDirectory.return_value.name
-    )
+    # FIXME
```

Comment:
I looked up if I could fix this one. It seems to be not so easy, because the temporary folder is deleted by `git_clone` (line 224), so when you want to make the assertion it is impossible to re-create the `git.Repo` object...

Reply:
Just to make sure I was clear: the problem is due to to the fact that `git_clone` does not take a string (path) but a repo object

Reply:
> Just to make sure I was clear: the problem is due to to the fact that git_clone does not take a string (path) but a repo object

Didn't understood why is this. Could you please clarify?

Reply:
ok, I just used `mocked_git_push.assert_called_once`. I guess this is ok.